website/conferences/views.py

```diff
@@ -158,7 +158,7 @@ def _render_conference_node(node, idx, conf):
         )
     except StopIteration:
         download_url = ''
-        download_count = 0
+        view_and_download = record.visit
```

Comment:
This is the main reason the build is failing, but there's also an API test that doesn't expect `visit` to be returned in `.to_storage`